sdk/build.gradle

```diff
@@ -10,7 +10,7 @@ apply plugin: "org.jetbrains.dokka"
 apply plugin: 'io.radar.mvnpublish'
 
 ext {
-    radarVersion = '3.9.0'
+    radarVersion = '3.9.2'
```

Comment:
Should be `3.9.1`, no?

Reply:
An earlier PR has `3.9.1` already but if we're getting this in first I can change this. Tested on Android simulator and device :+1: